decentraland/unity-explorer
# Chore: analytics identify on re-login

## What does this PR change?

Handle case of re-login for Identify analytics.

## How to test the changes?

1. Launch the explorer
2. Verify that "happy path" is not broken

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
